.github/docs_pr_change_template.md

```diff
@@ -0,0 +1,10 @@
+## Description
+
+<!-- Describe the changes made in this PR -->
+
+## Checklist
+
+- [ ] This change matches the [docs style guide](https://goteleport.com/docs/contributing/documentation/style-guide/), and the terms used in this change correspond to those in [Teleport Core Concepts](https://goteleport.com/docs/core-concepts)
```

Comment:
Based on my conversations with Sasha and Xin, this seems like more of the responsibility of the docs team. I don't think it hurts to capture it here (engineering should make an effort) but we don't have the wholistic view that you do.

Reply:
We've also talked about the docs team reviewing all docs PRs, and were told that was unrealistic. It can't be our sole responsibility if we're also not reviewing every docs change.